api/api.go

```diff
@@ -177,28 +176,36 @@ func (a *API) HealthCheck(w http.ResponseWriter, r *http.Request) error {
 	})
 }
 
-func WithInstanceConfig(ctx context.Context, smtp conf.SMTPConfiguration, config *conf.Configuration, instanceID uuid.UUID) (context.Context, error) {
+func WithInstanceConfig(ctx context.Context, config *conf.Configuration, instanceID uuid.UUID) (context.Context, error) {
```

Comment:
ok I think I get what is going on here - but I am also a little confused. Why aren't we taking the global mailers smtp configuration?

Reply:
Instead of having SMTP be a special flower, it is merged just like the external provider configs. See https://github.com/netlify/gotrue/pull/139/commits/f41e4ba7047f554e50933305de280fb464edd5bf#diff-604f4192deea5adf9c159e5f88f64630R206